BlockchainSdk/WalletManagers/Ethereum/EthereumNetworkService.swift

```diff
@@ -164,6 +164,17 @@ class EthereumNetworkService: MultiNetworkProvider {
         
         return blockchairProvider.findErc20Tokens(address: address)
     }
+
+    func getAllowance(from: String, to: String, contractAddress: String) -> AnyPublisher<String, Error> {
+        providerPublisher {
+            $0.getAllowance(from: from, to: to, contractAddress: contractAddress)
+                .tryMap {[weak self] in
+                    guard let self = self else { throw WalletError.empty }
+
+                    return try self.getResult(from: $0) }
```

Comment:
```suggestion
                .tryMap { [weak self] in
                    guard let self = self else { throw WalletError.empty }

                    return try self.getResult(from: $0)
                  }
```

скобочки бы поправить, отступ у верхней, и последнюю :)